$NetBSD: patch-aa,v 1.19 2002/09/13 15:53:10 tron Exp $

If LSOF_MORE_SECURE is non-zero, compile lsof with security, and
only let unprivileged (non-root) users view their own open files

--- dialects/n+obsd/machine.h	2001/04/30 09:23:38	1.1
+++ dialects/n+obsd/machine.h	2001/04/30 10:06:24
@@ -373,7 +373,9 @@
  * (the one that its user logged on with) of the lsof process.
  */
 
-/* #define	HASSECURITY	1 */
+#if LSOF_MORE_SECURE
+#define	HASSECURITY	1
+#endif
 
 
 /*
